libs/portals/core/src/types/portalCore.ts

```diff
@@ -57,6 +57,21 @@ export interface PortalNavigationItem {
    * Active state if current path is exact match
    */
   activeIfExact?: boolean
+
+  /**
+   * Service provider
+   */
+  serviceProvider?: string
+
+  /**
+   * Parent
+   */
+  parent?: PortalNavigationItem
+
+  /**
+   * Is key item? Stafrænt Íslands product
```

Comment:
~ What does it mean to be a key item?

Reply:
I believe this was added when the initial work was started but is not being used now.